### PR TITLE
alert and avatar component

### PR DESCRIPTION
**Pull request by sumitk-singh**
please review the alert and avatar component.

### CODE REVIEW

File: components/Alert.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+/* *** */
+
+
+/* alert */
+
+.alert-box {
+    margin: 8px;
+    padding: 15px;
```
Comment (Sahil25061999) [inline components/Alert.css]: try using rem , rest of the code looks fine can be merge

Comment (sumitk-singh) [reply components/Alert.css]: sure

**Review by zoyatanweer — APPROVED** [review]

File: components/Avatar.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+/* *** */
+
+.avatar {
+    border-radius: 50%;
+}
+
+.xl-size {
+    width: 200px;
```
Comment (Megha-Joshi) [inline components/Avatar.css:22]: use rem in place of px.

File: docs-page.html [hunk 3]
```diff
@@ -66,12 +66,85 @@ <h2 class="sidebar-h2">Components</h2>
...
+                <h1>Alerts</h1>
+                <h3>
+                    Alerts are used to attract user's attention for important information without interrupting the user's flow.
+                </h3>
+                <h2>Example of Alert</h2>
+                <p>Alerts are available in 8 types- primary alert, secondary alert, success alert, danger alert, warning alert, info alert, light alert, dark alert</p>
+                <p>To use this alert just use class name <span class="highlight-text">alert-box</span>and also add class according to alert type- <span class="highlight-text">alert-primary, alert-secondary, alert-success,
+                        alert-danger, alert-warning, alert-info, alert-light, alert-dark</span>( For e.g <span class="highlight-text">class="alert-box alert-primary"</span>)</p>
+                <p> You can copy html part from below code snippet.</p>
+                <div class="alert-example">
```
Comment (Megha-Joshi) [inline docs-page.html:79]: try to use semantic elements in place of div

File: docs-page.html [hunk 4]
```diff
@@ -66,12 +66,85 @@ <h2 class="sidebar-h2">Components</h2>
...
+                    <div class="alert-box alert-primary">
+                        A primary alert example with a text link
+                    </div>
+
+                    <div class="alert-box alert-secondary">
+                        A secondary alert example!!!!
+                    </div>
+                    <div class="alert-box alert-success">
```
Comment (Megha-Joshi) [inline docs-page.html:87]: give proper spacing

**Review by Megha-Joshi — COMMENTED** [review]
suggested some changes

**Pull request merged**